File: java/day01.java
package java;

public class day01{
    public static void main(String[] args) {
        float a = 87f;
        float b = 12f;
        float c = 123f;
        a = (int)a;
        b = (int)b;
        c = (int)c;

        float add = Math.round((a + b)*10)/10;
        float sub = Math.round((a - b)*10)/10;
        float mult = Math.round((a * b)*10)/10;
        double divi = a / b;
        double pyt = Math.sqrt((Math.pow(a, 2) + Math.pow(b, 2)));

        System.out.println("The sum of these numbers is " + add);
        System.out.println("The difference of these numbers is " + sub);
        System.out.println("The product of these numbers is " + mult);
        System.out.println("The quotient of these numbers is " + divi);
        System.out.println("Using a and b in the pythagorean equation results in " + pyt);


    }
    
};

File: python/day01.py
import math

def myMaths(a, b):
    sum = "The sum of "+ str(a) + " and " + str(b) + " is "+ str(a+b);
    difference = ". The difference of "+ str(a) + " and " + str(b) + " is "+ str(a-b);
    product = ". The product of "+ str(a) + " and " + str(b) + " is "+ str(a*b);
    quotient = ". The quotient of "+ str(a) + " and " + str(b) + " is "+ str(a/b);
    module = ". The module of "+ str(a) + " and " + str(b) + " is "+ str(a%b);
    power = ". The result of "+ str(a) + " to the power of " + str(b) + " is "+ str(math.pow(a, b));
    pythagorean = ". The result of the pythagorean therom with "+ str(a) + " and " + str(b) + " is "+ str(math.sqrt((math.pow(a, 2) + math.pow(b, 2))));
    print(sum + difference + product + quotient + module + power + pythagorean);


name = input("What is your name? ")
print("hello, "+name);
num1 = float(input("What is your first number? "));
num2 = float(input("What is your second number? "));
myMaths(num1, num2);



